StringTokenFormatter/TokenReplacer.cs

```diff
@@ -90,23 +96,27 @@ public string FormatFromContainer(SegmentedString segmentedString, ITokenValueCo
             if (segmentedString == null) throw new ArgumentNullException(nameof(segmentedString));
             if (container == null) throw new ArgumentNullException(nameof(container));
 
-            StringBuilder sb = new StringBuilder();
+            var sb = new StringBuilder();
```

Comment:
What's changed here? the two look equivalent, am I missing something?

Reply:
just '''var'''.